financialaid/migrations/0003_added_timestamps.py

```diff
@@ -4,7 +4,7 @@
 
 import datetime
 from django.db import migrations, models
-
+import pytz
```

Comment:
This was necessary because it was causing warnings in the selenium tests, which explicitly run all migrations to populate an initial database